apps/onboarding/src/config/rules.config.ts

```diff
@@ -0,0 +1,21 @@
+import { registerAs } from '@nestjs/config'
+
+function getConfig(config: string | undefined) {
+  try {
+    return JSON.parse(config)
+  // tslint:disable-next-line
+  } catch { }
```

Comment:
Let's actually move the responsibility of parsing and validating a config to the rules themselves.
There's a method in there called 'validateConfig` defined on the `Rule` interface lets change that to:
```
parseConfig(rawConfig: string): TRuleConfig
```

And have that take the string and return a valid config.
It shouldn't handle both json parsing and runtime type verifications using [io-ts](https://github.com/gcanti/io-ts) or in the case of no-config rules just do a noop, as it's currently doing.

And that part of the rules config should throw an error when an invalid config is passed.